Configuration.props

```diff
@@ -56,7 +56,7 @@
     <AllSupported32BitTargetAndroidAbis>armeabi;armeabi-v7a;x86</AllSupported32BitTargetAndroidAbis>
     <AllSupported64BitTargetAndroidAbis>arm64-v8a;x86_64</AllSupported64BitTargetAndroidAbis>
     <AllSupportedTargetAndroidAbis>$(AllSupported32BitTargetAndroidAbis);$(AllSupported64BitTargetAndroidAbis)</AllSupportedTargetAndroidAbis>
-    <XABuildToolsVersion>25.0.1</XABuildToolsVersion>
+    <XABuildToolsVersion>26.0.0-rc1</XABuildToolsVersion>
```

Comment:
`build-tools_r26.0.0-rc1-macosx.zip` doesn't exist [causing a failure](https://jenkins.mono-project.com/job/xamarin-android-pr-builder/738/):

    Unable to download URL `https://dl-ssl.google.com/android/repository/build-tools_r26.0.0-rc1-macosx.zip` to `/Users/builder/android-archives/build-tools_r26.0.0-rc1-macosx.zip`: 404 (Not Found)